src/tasks/components/NewTaskScheduler/TaskScheduler.tsx

```diff
@@ -0,0 +1,181 @@
+import React, {FC, ChangeEvent, useCallback, useEffect, useState} from 'react'
+
+// Components
+import {
+  Button,
+  ButtonShape,
+  ButtonType,
+  Columns,
+  ComponentColor,
+  Form,
+  Grid,
+  IconFont,
+  Input,
+  Page,
+  SelectGroup,
+} from '@influxdata/clockface'
+import TaskSchedulerFormField from 'src/tasks/components/NewTaskScheduler/TaskSchedulerFormField'
+import ScriptSelector from 'src/tasks/components/NewTaskScheduler/ScriptSelector'
+
+// Utils
+import {debouncer} from 'src/dataExplorer/shared/utils'
+import {SafeBlankLink} from 'src/utils/SafeBlankLink'
+
+// Types
+import {RemoteDataState, TaskOptions, TaskSchedule} from 'src/types'
+
+import 'src/tasks/components/NewTaskScheduler/TaskScheduler.scss'
+
+const getScripts = require('src/client/scriptsRoutes').getScripts
+
+interface Props {
+  taskOptions: TaskOptions
+  onChangeScheduleType: (schedule: TaskSchedule) => void
+  onChangeInput: (e: ChangeEvent<HTMLInputElement>) => void
+}
+
+const TaskScheduler: FC<Props> = ({
+  taskOptions,
+  onChangeScheduleType,
+  onChangeInput,
+}) => {
+  const [loading, setLoading] = useState(RemoteDataState.NotStarted)
+  const [scripts, setScripts] = useState([])
+  const [searchTerm, setSearchTerm] = useState('')
+  const [selectedScript, setSelectedScript] = useState<any>({})
+  const [taskName, setTaskName] = useState('')
+
+  const handleGetScripts = useCallback(async (name: string = '') => {
+    try {
+      setLoading(RemoteDataState.Loading)
+      const resp = await getScripts({query: {limit: 250, name}})
+
+      if (resp.status !== 200) {
+        throw new Error(resp.data.message)
+      }
+
+      setScripts(resp.data.scripts)
+      setLoading(RemoteDataState.Done)
+    } catch (error) {
+      setLoading(RemoteDataState.Error)
+      console.error({error})
```

Comment:
🙅 `console.error` does nothing for us. It spits out an error in the user's console, and provides no information to them. This should be replaced (in the thunk layer) with either a honeybadger call and/or notification that there was an error fetching scripts. We shouldn't add any `console.error` calls under the majority of circumstances anymore